app/models/bid_status_presenter_factory.rb

```diff
@@ -34,7 +34,9 @@ def future_message
   end
 
   def over_winning_bidder_message
-    if auction.pending_acceptance?
+    if auction.accepted? && auction.accepted_at.nil?
```

Comment:
yes, this is how we know something was accepted but the payment URL is missing. Kinda weird, perhaps something to change in a future PR. Added issue here: https://github.com/18F/micropurchase/issues/1222